src/Microsoft.AspNetCore.Mvc.Razor/Internal/ViewComponentTagHelperDescriptorFactory.cs

```diff
@@ -56,7 +57,6 @@ public IEnumerable<TagHelperDescriptor> CreateDescriptors(string assemblyName)
                     StringComparison.Ordinal));
 
             var tagHelperDescriptors = viewComponentDescriptors
-                .Where(descriptor => !descriptor.Parameters.Any(parameter => parameter.ParameterType.GetTypeInfo().IsGenericType))
```

Comment:
But that isn't even a view component - view components can't be generic - so it can't possibly be available as a view component tag helper.